src/visit_vtk/full/vtkAxisDepthSort.h

```diff
@@ -44,15 +46,15 @@ class VISIT_VTK_API vtkAxisDepthSort : public vtkPolyDataAlgorithm
 
   protected:
                                   vtkAxisDepthSort();
-    virtual                      ~vtkAxisDepthSort() {;};
+    virtual                      ~vtkAxisDepthSort() {;}
 
     int                           RequestData(vtkInformation *,
                                       vtkInformationVector **,
                                       vtkInformationVector *) override;
 
   private:
-                                  vtkAxisDepthSort(const vtkAxisDepthSort &);
-                                  void operator=(const vtkAxisDepthSort &);
+                                  vtkAxisDepthSort(const vtkAxisDepthSort &) = delete;
+                                  void operator=(const vtkAxisDepthSort &) = delete;
```

Comment:
Is `private` still necessary if using `= delete;`

Reply:
Unclear to me. The change to this file was to be consistent with other files that had previously been using 'VTK_DELETE' and needed to be changed to 'delete'.

Reply:
Ah, ok. I think I was misunderstanding then. Does `= delete` actually remove them from the vtable? If so, that is indeed something different than making the private (they're in the vtable but just not callable outside). Honestly, I am just guessing here as this is the first time I've ever seen this construct.

Reply:
Introduced in C++11, to avoid implicit declaration of a copy constructor or assignment operator.
https://en.cppreference.com/w/cpp/language/copy_constructor